index.html

```diff
@@ -0,0 +1,195 @@
+                            <p class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127">
+                                Listen to the speakers from various countries about the messages.
+                            </p>
+                        </li>
+                        <li class="p-3 col-md-2 row m-1 justify-content-center align-items-center">
+                            <img src="img/program_icon_02.png" alt="podium" class="pt-md-2 col-md-auto col-2 program-icon"/>
+                            <h6 class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2">GDG content</h6>
+                            <p class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127">
+                                Google technologies in areas like Android, Flutter and so much more!
+                            </p>
+                        </li>
+                        <li class="p-3 col-md-2 row m-1 justify-content-center align-items-center">
+                            <img src="img/program_icon_03.png" alt="podium" class="pt-md-2 col-md-auto col-2 program-icon"/>
+                            <h6 class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2">GDG Forum</h6>
+                            <p class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127">
+                                Share your thoughts and opinions with experts for each topic.
+                            </p>
+                        </li>
+                        <li class="p-3 col-md-2 row m-1 justify-content-center align-items-center">
+                            <img src="img/program_icon_04.png" alt="podium" class="pt-md-2 col-md-auto col-2 program-icon"/>
+                            <h6 class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2">Workshops</h6>
+                            <p class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127"> 
+                                Full-day workshops, hands-on experiences, and debugging events.
+                            </p>
+                        </li>
+                        <li class="p-3 col-md-2 row m-1 justify-content-center align-items-center">
+                            <img src="img/program_icon_05.png" alt="podium" class="pt-md-2 col-md-auto col-2 program-icon"/>
+                            <h6 class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2">Networking</h6>
+                            <p class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127">Get opportunities to network with Coders around the world.</p>
+                        </li>
+                    </ul>
+                </div>
+                <div>
+                    <a class="btn-lg text-light" href="#" role="button">Join devfest 2023</a>
+                </div>
```

Comment:
- [ ] On mobile, this button/ link needs to look like a button and not a "stand alone link". Kindly implement this.

**This is the desired design on mobile:**

![image](https://user-images.githubusercontent.com/86318284/229051619-881bef27-c09c-4f16-96d3-96b990efcefa.png)

**This is how your design looks like:**

![image](https://user-images.githubusercontent.com/86318284/229051738-c2bf180f-439b-41f2-aebb-75707defd556.png)

Reply:
@Nemwel-Boniface Cool. This was implemented already.

Reply:
Well resolved :heavy_check_mark: 
:heavy_multiplication_x:    New issue introduced as well. On desktop view, the "Join devfest 2023" is missing. :heavy_multiplication_x: